src/components/NewMessage/NewMessageAbsenceInfo.vue

```diff
@@ -15,6 +15,13 @@
 				disable-tooltip />
 		</template>
 		<h4 class="absence-reminder__caption">{{ userAbsenceCaption }}</h4>
+		<div v-if="hasReplacementUser" class="absence-reminder__replacement">
+			<p>{{ t('spreed','Replacement: ') }}</p>
```

Comment:
For plain, one-word lines it is better to leave an explanation for translators
```suggestion
			// TRANSLATORS An acting person during the period of absence of the main contact
			<p>{{ t('spreed','Replacement: ') }}</p>
```